overcooked_ai_py/agents/benchmarking.py

```diff
@@ -379,8 +381,19 @@ def update(t = 1.0):
         display(out, t)
 
     # EVENTS VISUALIZATION METHODS #
-    
+        
     @staticmethod
-    def events_visualization(trajs, traj_index):
-        # TODO
-        pass
+    def events_visualization(trajs, traj_index=0, ipython=False, chart_settings=None):
+        """
+        Displays chart with visualization of events (when items pickups happened etc.)
+        ipython - chooses between opening chart
+        in default browser or below ipython cell (for ipython=True)
+        chart_settings - json with various chart settings that overwrittes default ones
+        for more info see create_chart_html function with comments aboutr default chart settings
+        """
+        events = extract_events(trajs, traj_index)
+        html = create_chart_html(events, chart_settings)
+        if ipython:
+            run_html_in_ipython(html)
+        else:
+            run_html_in_web(html)
```

Comment:
I would also recommend writing 1-2 unit tests for each new piece of code you write. It's a good habit to get into, and even if it seems silly as the code you'd be testing is very simple or high level, it's a great way to ensure that downstream changes don't break upstream things like visualization tools

Reply:
Unit tests added